website/careers/index.html

```diff
@@ -0,0 +1,320 @@
+        {{ _('We’re committed to moving the code base over to using primarily web technologies, and you’ll be part of this effort. Contrary to many opportunities elsewhere, you’ll have the opportunity to utilize the most cutting edge developments in the browser engine.') }}
+      </p>
+      <p class="font-lg tracking-wider leading-loose mb-6 p-links-blue">
+        {{ _('As an engineer at Thunderbird, you’ll be working with existing staff and community members from all over the globe. We work very much in the open, and you’ll be able to show your skills to the world both on the code level and as a concrete results in improvements on the product.') }}
+      </p>
+      <p class="font-lg tracking-wider leading-loose mb-10 p-links-blue">
+        {{ _('This is a <strong>remote</strong>, <strong>full time</strong> position. You are expected to have excellent written communication skills and coordinate your work over email, video conferencing, IRC, and Bugzilla.') }}
+      </p>
+    </aside>
+    <aside>
+      <h4 class="subheader-section">{{ _('You could end up working on:') }}</h4>
+      <ul class="font-md tracking-wider leading-loose mb-10 p-links-blue">
+        <li>
+          {{ _('Localization, moving Thunderbird to using Fluent.') }}
+        </li>
+        <li>
+          {{ _('The ongoing effort of converting the user interface from using XUL to HTML.') }}
+        </li>
+        <li>
+          {{ _('Converting filtering code implementation to JavaScript, and async operation.') }}
+        </li>
+        <li>
+          {{ _('Compose pipeline modernization.') }}
+        </li>
+        <li>
+          {{ _('Integrating some of the best parts of the more popular add-ons into core.') }}
+        </li>
+        <li>
+          {{ _('...and <a href="http://lists.thunderbird.net/pipermail/maildev_lists.thunderbird.net/2019-November/002024.html">much more</a>.') }}
+        </li>
+      </ul>
+    </aside>
+    <aside>
+      <h4 class="subheader-section">{{ _('We’re looking for someone with:') }}</h4>
+      <ul class="font-md tracking-wider leading-loose mb-10 p-links-blue">
+        <li>
+          {{ _('Passion for open source, and open standards.') }}
+        </li>
+        <li>
+          {{ _('Good understanding of web technologies.') }}
+        </li>
+        <li>
+          {{ _('Proven experience developing with JavaScript/HTML/CSS.') }}
+        </li>
+        <li>
+          {{ _('Knowledge of C/C++.') }}
+        </li>
+        <li>
+          {{ _('Good problem solving skills.') }}
+        </li>
+      </ul>
+      <a href="#apply" class="btn-join self-start">{{ _('How to Apply')}}</a>
+    </aside>
+  </section>
+
+  <section class="pt-10 pb-10 flex flex-col max-w-6xl mx-auto justify-between pl-8 pr-8">
+    <aside>
+      <h3 class="header-section">
+        <span>{{ svg('bookmark') }}</span>
+        <aside class="flex flex-col">
+          {{ _('Add-ons Coordinator')}}
```

Comment:
Ryan and I agree a better name for this would be "Add-on Support Engineer" so lets change that.

Reply:
I think this is a reasonable update. Let's change it.